aspnetcore/web-api/index.md

```diff
@@ -71,11 +71,14 @@ The following sections describe convenience features added by the attribute.
 
 ### Automatic HTTP 400 responses
 
-Validation errors automatically trigger an HTTP 400 response. The following code becomes unnecessary in your actions:
+Model validation errors automatically trigger an HTTP 400 response. Consequently, the following code becomes unnecessary in your actions:
 
 [!code-csharp[](define-controller/samples/WebApiSample.Api.Pre21/Controllers/PetsController.cs?name=snippet_ModelStateIsValidCheck)]
 
-Use <xref:Microsoft.AspNetCore.Mvc.ApiBehaviorOptions.InvalidModelStateResponseFactory> to customize the output of the resulting response.
+Disabling this behavior is useful when your action:
+
+* Can recover from a model validation error.
+* Should return a customized response using <xref:Microsoft.AspNetCore.Mvc.ApiBehaviorOptions.InvalidModelStateResponseFactory>.
```

Comment:
This bullet is now backwards. The `InvalidModelStateResponseFactory` is used when `[ApiController]` applies or the `ModelStateInvalidFilter` has been manually configured. The previous sentence was correct.

But, in addition, users may wish to remove `[ApiController]` or set `ApiBehaviorOptions.SuppressModelStateInvalidFilter` to `true` if their action can recover from a model validation error.